metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -304,6 +304,17 @@ object ServerCommands {
     "[uri], the uri of the worksheet that you'd like to copy the contents of."
   )
 
+  val InsertInferredType = new Command(
+    "insert-inferred-type",
+    "Insert inferred type of a value",
+    """|Whenever user chooses code action to insert inferred type this command is later run to 
+       |calculate the type and insert it in a correct location.
```

Comment:
```suggestion
    """|Whenever a user chooses code action to insert the inferred type this command is later ran to 
       |calculate the type and insert it in the correct location.
```

Reply:
Fun fact, I think it's the most common English error for people from Slavic languages. We never really need to insert anything like `the`/`a`/`an` and it's so embedded that it's often hard to see the error even.

Reply:
Fixed!